srcpkgs/snooze-cron-lite/files/snooze-daily/finish

```diff
@@ -0,0 +1 @@
+#!/usr/bin/touch /var/cache/snooze/daily
```

Comment:
This is a cute hack, but will also touch the file in /etc, which I'd like to avoid.  Please use /bin/sh too.